packages/test/test-end-to-end-tests/src/test/SummarizeFetchValidation.spec.ts

```diff
@@ -369,10 +369,9 @@ describeFullCompat("Summarizer fetches expected number of times", (getTestObject
 		let uploadSummaryUploaderFunc = containerRuntime.storage.uploadSummaryWithContext;
 		const func = async (summary: ISummaryTree, context: ISummaryContext) => {
 			uploadSummaryUploaderFunc = uploadSummaryUploaderFunc.bind(containerRuntime.storage);
-			const response = await uploadSummaryUploaderFunc(summary, context);
 			// Close summarizer so that it does not submit SummaryOp
 			summarizer.close();
-			return response;
```

Comment:
This seems hacky. If the purpose of the test is to just fail summaries, you could override the `summarize` function instead and just throw an error in it. You could also throw an error in this function itself without doing any of the other stuff. That would make it much clear what is happening.

Reply:
The goal was to simply close the summarizer before the upload happens. I could also close and let the upload happen. That would not affect anything.  Or I could special case for ODSP and force fetch from latest on the next summarize.  That would also do it.